inginious/frontend/plugins/manual_scoring/static/js/manual_scoring_main.js

```diff
@@ -29,35 +27,74 @@ function save(rubric) {
     });
 }
 
-function loadFeedBack() {
-    const feedbackContent = getHtmlCodeForFeedback();
-    const feedbackType = getTextBoxTypeBasedOnResult();
-    const message = new MessageBox(FEEDBACK_FIELD_ID, feedbackContent, feedbackType, false);
-    message.deleteCloseButton();
+function updateScoreOnInfo(score) {
+    const grade = new Score(GRADE_ID, score);
+    grade.updateScore();
+    grade.changeColor();
 }
 
+function addSaveFunctionToSaveButton(rubric) {
+    $(`#${SAVE_BUTTON_ID}`).click(function () {
+        save(rubric);
+    });
+}
 
-function addToggleBehaviorToProblemDescription() {
-    $(`#${TASK_NAME}`).click(function () {
-        $(`#${TASK_DESCRIPTION_TEXT_ID}`).collapse("toggle");
+function previewCode() {
+    const contentDanger = "Something went wrong";
+    const txtComment = document.getElementById(COMMENTS_TEXT_AREA_ID);
+    $(`#${PREVIEW_TAB_ID}`).click(function () {
+        $.ajax("/api/preview_content", {
+            method: "POST",
+            data: {
+                content: txtComment.value
+            },
+            success: function (data) {
+                $(`#${PREVIEW_AREA_ID}`).html(data);
+            },
+            error: function (request, status, error) {
+                const message = new MessageBox(RESPONSE_FIELD_ID, contentDanger, "danger");
+            }
+
+        })
     });
 }
 
-function addSaveFunctionToSaveButton(rubric) {
-    $(`#${SAVE_BUTTON_ID}`).click(function () {
-        save(rubric);
+function sendManualScoringAnalytics() {
+    $.post('/api/analytics/', {
+        service: {
+            key: "manual_scoring_review",
+            name: "Manual Scoring - review"
```

Comment:
review? Is this the admin page? probably something like `manual scoring creation` or something like that